bowtie/_cli.py

```diff
@@ -484,21 +492,37 @@ async def _info(image_names: list[str]):
                 click.echo("  ❗ (error): startup failed")
                 continue
 
-            click.echo(
-                "\n".join(
-                    f"{k}: {json.dumps(v, indent=2)}"
-                    for k, v in sorted(
-                        implementation.metadata.items(),
-                        key=lambda kv: (
-                            kv[0] != "name",
-                            kv[0] != "language",
-                            kv[0] != "version",
-                            kv[0] == "dialects",
-                            kv[0],
-                        ),
-                    )
-                ),
-            )
+            metadata = {
+                k: v
+                for k, v in sorted(
+                    implementation.metadata.items(),
+                    key=lambda kv: (
+                        kv[0] != "name",
+                        kv[0] != "language",
+                        kv[0] != "version",
+                        kv[0] == "dialects",
+                        kv[0],
+                    ),
+                )
+            }
+            if format == "json":
+                click.echo(json.dumps(metadata, indent=2))
+            else:
+                click.echo(
+                    "\n".join(
```

Comment:
I think you also can make do without calculating this a second time here, you should just be able to at least partially reuse `metadata` here without recalculating it from` implementation.metadata`. I.e. something like:

`"\n".join(f"{k}: {json.dumps(v, indent=2)}" for k, v in metadata.items()}` probably should do it though I haven't double checked.